migrations/20160821173914_records_primary_id_to_source_trial_id.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+exports.up = (knex) => (
+  knex.schema
+    .table('records', (table) => table.renameColumn('primary_id', 'source_trial_id'))
```

Comment:
Do you know if this `primary_id` is being used anywhere? In the code we're getting the primary ID by using `identifiers[trial.primary_source_id]` (should be changed after this is merged).